src/AnyAddress.h

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Data.h"
+#include "PublicKey.h"
+
+#include <TrustWalletCore/TWCoinType.h>
+#include <TrustWalletCore/TWPublicKey.h>
+#include <TrustWalletCore/TWData.h>
+#include <string>
+
+namespace TW {
+
+class AnyAddress {
```

Comment:
Everything is public, use the struct keyword instead

Reply:
I would like to leave it as-is. To my taste, `struct` keyword is better to use for plain C structs without methods/etc